### ISSUE TITLE
Updates in Shiny duplicate TnTBoard/Genome view

### ISSUE THREAD

**Issue by jrboyd** [Marlin-Na/TnT#3]
Hello,
Thank you for your work on this package and especially for including Shiny bindings.  I'd like to have the ability to adjust display settings for tracks via Shiny inputs but have an issue where an entirely new TnTBoard/Genome is appended below the existing view instead of replacing/updating the existing one.  

I've modified one of your examples to include a toggle for Area/LineTracks to demonstrate:

````
if (interactive() && require(shiny)) {
  ui <- fluidPage(fluidRow(
    column(width = 2, {
      "A Simple Example Here"
      checkboxInput("Fill", "Fill?", value = T)
    }),
    column(width = 10, {
      TnTOutput("out")
    })
    
  ))
  server <- function (input, output) {
    signal = c(rep(0, 5), table(round(rnorm(100, mean = 10))), rep(0, 5))
    gr <- GRanges("chr12", IRanges(1:length(signal), 1:length(signal)))
    gr$score = signal

    
    re.btrack <- reactive({
      
      
      if(input$Fill){
        AreaTrack(gr, value = gr$score, color = "blue")
      }else{
        LineTrack(gr, value = gr$score, color = "blue")
      }
    })
    output$out <- renderTnT({
      TnTGenome(re.btrack())
    })
  }
  shinyApp(ui = ui, server = server)
}
````

**Comment by Marlin-Na** [comment]
Thanks for trying out the package and reporting this !

Yes, it is a bug and should be fixed in version 1.0.1 (bioc release) and 1.1.1 (bioc devel). It will be available for `biocLite` in approximately 24 hours. Or you can install from github via `devtools::install_github("marlin-na/TnT")`.

If you find out any other problem with the shiny binding, please let me know.

Thanks!

**Comment by jrboyd** [comment]
That was fast! looks fixed.
